### ISSUE TITLE
Add argument hit to annotations

### ISSUE THREAD

**Issue by KoBeWi** [godotengine/godot-proposals#10681]
### Describe the project you are working on

Game using GDScript.

### Describe the problem or limitation you are having in your project

There are many `@export` annotations with various arguments and sometimes it's difficult to remember what takes what.

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

Make annotations show argument hint when writing them, similar to methods.
![image](https://github.com/user-attachments/assets/3df78869-637f-4d6b-bcea-5e638fae6525)

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

The implementation would be similar to method's argument hint I presume.

### If this enhancement will not be used often, can it be worked around with a few lines of script?

No.

### Is there a reason why this should be core and not an add-on in the asset library?

It's usability enhancement for built-in code editor.

**Comment by HolonProduction** [comment]
One of those things that make you wonder why they don't exist yet.

I wouldn't put `void` in front of it and instead use `@export_custom` as name.

**Comment by KoBeWi** [comment]
> I wouldn't put void in front of it and instead use @export_custom as name.

Yeah I just made `func export_custom` and copied the tooltip to make the mockup.
